cookbook/core/control_flow/chain_tasks.py

```diff
@@ -1,34 +1,30 @@
 """
-Chain Flyte Tasks
------------------
+Chain Flyte Entities
+--------------------
 
-Data passing between tasks need not always happen through parameters.
-Flyte provides a mechanism to chain tasks using the ``>>`` operator and the ``create_node`` function.
-You may want to call this function to specify dependencies between tasks that don't consume or produce outputs.
+Data passing between tasks need not necessarily happen through parameters.
+In such a case, if you want to explicitly construct the tasks' dependency, flytekit provides a mechanism to chain tasks using the ``>>`` operator and ``create_node`` function.
 
 In this example, let's enforce an order for ``read()`` to happen after ``write()``.
 """
-
+# %%
```

Comment:
nit: can we have a new line before we start the content block?

Reply:
Done.